bin/src/framebuffer.rs

```diff
@@ -36,6 +43,14 @@ impl Framebuffer<'_>{
         }
     }
 
+    pub fn get_color_from_state(state: DisplayState) -> Color565 {
+        match state {
+            DisplayState::Paused => Color565::White,
+            DisplayState::Recording => Color565::Green, 
+            DisplayState::WarningDetected => Color565::Red,
+        }
+    }
+
```

Comment:
nitpick: idiomatically i think this should be an `impl From<DisplayState> for Color565 {}`, and then we could just say `display_color = state.into()` in the recv block